### ISSUE TITLE
Installing from pip gives encoding error

### ISSUE THREAD

**Issue by ashwin** [spantaleev/sftpman-gtk#2]
```
$ sudo pip install sftpman-gtk
Downloading/unpacking sftpman-gtk
  Downloading sftpman-gtk-0.4.1.tar.gz
  Running setup.py (path:/tmp/pip_build_root/sftpman-gtk/setup.py) egg_info for package sftpman-gtk
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip_build_root/sftpman-gtk/setup.py", line 12, in <module>
        import sftpman_gtk
      File "sftpman_gtk/__init__.py", line 8
    SyntaxError: Non-ASCII character '\xc2' in file sftpman_gtk/__init__.py on line 8, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip_build_root/sftpman-gtk/setup.py", line 12, in <module>

    import sftpman_gtk

  File "sftpman_gtk/__init__.py", line 8

SyntaxError: Non-ASCII character '\xc2' in file sftpman_gtk/__init__.py on line 8, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
```

System is Ubuntu 14.04 and Python 2.7.6.

**Comment by brain-pilot** [comment]
Exact same error here, have you found a solution yet?

**Comment by Saareem** [comment]
I get the same error. I know what is the culprit: there's unicode copymark (c) sign on the 
**init**.py file. 

``` python
__copyright__ = 'Copyright © 2011-%d %s' % (datetime.date.today().year, __author__)
```

Maybe declaring # -_\- coding: utf-8 -_\- on the beginning of the file could help. Or just turn it into (c)

EDIT:
Editing the init.py -file as I suggested (adding # -_\- coding: utf-8 -_\- on the top)  did the trick. 

I cloned this repository with
`git clone https://github.com/spantaleev/sftpman-gtk`
then edited as suggested and finally run
`sudo setup.py install`
in sftpman-gtk directory

Then you have to make /mnt/sshfs directory or the program will not work
`sudo mkdir /mnt/sshfs`
And you are good to go.

**Comment by spantaleev** [comment]
Fixed now and released as version `0.4.2`. Thank you!
